eeg_flow/tasks/epochs_evoked.py

```diff
@@ -0,0 +1,434 @@
+) -> None:
+    """Compute behav report and output metadata.
+    Parameters
+    ----------
+    %(participant)s
+    %(group)s
+    %(task)s
+    %(run)s
+    """
+    # prepare folders
+    _, derivatives_folder_root, _ = load_config()
+    fname_stem = get_fname(participant, group, task, run)
+    derivatives_folder = get_derivative_folder(
+        derivatives_folder_root, participant, group, task, run
+    )
+
+    # lock the output derivative files
+    # create locks
+    derivatives = [
+        derivatives_folder / (fname_stem + "_step9_a-metadata.csv"),
+        derivatives_folder / (fname_stem + "_step9_b-behav.txt"),
+        derivatives_folder / (fname_stem + "_step9_c1-cleaned-epo.fif"),
+        derivatives_folder / (fname_stem + "_step9_c2-drop-epochs.csv"),
+        derivatives_folder / (fname_stem + "_step9_c3-drop-channel-log.csv"),
+        derivatives_folder / (fname_stem + "_step9_d1-standard_evoked-ave.fif"),
+        derivatives_folder / (fname_stem + "_step9_d2-target_evoked-ave.fif"),
+        derivatives_folder / (fname_stem + "_step9_d3-novel_evoked-ave.fif"),
+    ]
+
+    locks = lock_files(*derivatives, timeout=timeout)
+
+    try:
+        _behav_prep_epoching(participant, group, task, run)
+    finally:
+        for lock in locks:
+            lock.release()
+        del locks
+    return
+
+
+@fill_doc
+def _behav_prep_epoching(
+    participant: str,
+    group: str,
+    task: str,
+    run: int,
+) -> None:
+    """Compute behav report and output metadata.
+    Parameters
+    ----------
+    %(participant)s
+    %(group)s
+    %(task)s
+    %(run)s
+    """
+    # prepare folders
+    _, derivatives_folder_root, _ = load_config()
+    fname_stem = get_fname(participant, group, task, run)
+    derivatives_folder = get_derivative_folder(
+        derivatives_folder_root, participant, group, task, run
+    )
+
+    # load previous steps
+    # # load raw_fit recording
+    raw = read_raw_fif(
+        derivatives_folder / (fname_stem + "_step7_preprocessed_raw.fif"),
+        preload=True,
+    )
+
+    events = find_events(raw, stim_channel="TRIGGER")
+
+    events_id = dict(standard=1, target=2, novel=3, response=64)
+    row_events = ["standard", "target", "novel"]
+
+    metadata, events, event_id = create_metadata(events, events_id, raw, row_events)
+    FNAME_METADATA = derivatives_folder / (fname_stem + "_step9_a-metadata.csv")
+    metadata.to_csv(FNAME_METADATA)
+
+    num_Hits, num_CorrectRejections, num_Misses, num_FalseAlarms = get_SDT_outcomes(metadata)
+
+    hits = metadata[metadata["response_type"] == "Hits"]
+    response_mean = round(hits["response"].mean(), 5)
+    response_std = round(hits["response"].std(), 5)
+
+    plot_RT(hits, fname_stem, derivatives_folder, response_mean, response_std)
+
+    get_indiv_behav(metadata, num_Hits, num_CorrectRejections, num_Misses, num_FalseAlarms, 
+                    fname_stem, derivatives_folder, response_mean, response_std)
+
+
+    # %%
```

Comment:
Avoid `# %%` or `#%%` (or any number of `#` after the `%`), they render as Jupyter cells in most IDEs and this is not appropriate since this cell is only composed of part of a function, i.e. this cell is broken.